_common.scss

```diff
@@ -30,10 +30,12 @@
     .mat-expansion-panel {
         .mat-expansion-panel-header {
             .mat-expansion-indicator {
+                transition: margin 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
                 margin-top: -8px;
             }
             &.mat-expanded {
                 .mat-expansion-indicator {
+                    transition: margin 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
```

Comment:
The transition under `mat-expanded` section isn't need; it's already inheriting the transition from above.